great_expectations/experimental/datasources/data_asset/data_connector/file_path_data_connector.py

```diff
@@ -247,7 +252,30 @@ def _generate_batch_spec_parameters_from_batch_definition(
 
         path = self._get_full_file_path(path=path)
 
-        return {"path": path}
+        return {FilePathDataConnector.FILE_PATH_BATCH_SPEC_KEY: path}
+
+    def _insure_regex_groups_include_data_reference_key(
```

Comment:
`ensure`

Reply:
@NathanFarmer Oops!  I did it again!  Fixing!  Thank you!